src/test/java/seedu/address/logic/commands/FindNumCommandTest.java

```diff
@@ -0,0 +1,96 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+import static seedu.address.testutil.TypicalPersons.ELLE;
+import static seedu.address.testutil.TypicalPersons.GEORGE;
+import static seedu.address.testutil.TypicalPersons.getTypicalNetConnect;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.PhoneContainsDigitsPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for
+ * {@code FindNumCommand}.
+ */
+public class FindNumCommandTest {
+    private final Model model = new ModelManager(getTypicalNetConnect(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalNetConnect(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        PhoneContainsDigitsPredicate firstPredicate = new PhoneContainsDigitsPredicate(
+                Collections.singletonList("first"));
+        PhoneContainsDigitsPredicate secondPredicate = new PhoneContainsDigitsPredicate(
+                Collections.singletonList("second"));
+
+        FindNumCommand findNumFirstCommand = new FindNumCommand(firstPredicate);
+        FindNumCommand findNumSecondCommand = new FindNumCommand(secondPredicate);
+
+        // same object -> returns true
+        assertEquals(findNumFirstCommand, findNumFirstCommand);
+
+        // same values -> returns true
+        FindNumCommand findNumFirstCommandCopy = new FindNumCommand(firstPredicate);
+        assertEquals(findNumFirstCommand, findNumFirstCommandCopy);
+
+        // different types -> returns false
+        assertNotEquals(1, findNumFirstCommand);
+
+        // null -> returns false
+        assertNotEquals(null, findNumFirstCommand);
```

Comment:
I think either change to `assertFalse(findNumFirstCommand.equals(null))`, or `assertNotEquals(findNumFirstCommand, null)`. Fixes code coverage in FindNumCommand line 44

Reply:
I realise this issue was introduced in PR #36, when all `assertFalse(command.equals(null))` was auto-reformatted to `assertNotEquals(null, command)`. The latter doesn't actually invoke the call of `command.equals(null)`